scripts/functions/requirements/elementary

```diff
@@ -14,7 +14,13 @@ requirements_elementary_define_libreadline()
 
 requirements_elementary_define_libssl()
 {
-  requirements_check libssl-dev
+  if [[ "${_system_name}" == "Elementary" && ("${_system_version}" == "5.0" || "${_system_version}" > "5.0") ]]
```

Comment:
> Should this logic maybe be in the requirements_debian_define_libssl() function in scripts/functions/requirements/debian or in a parallel function for elementary OS?
Since elementary and debian use different release numbers, release calendars and so on, I believe they should be separated, right? Debian routing should not decide wether eOS uses `libssl-dev` or `libssl1.0-dev`, for example.

As for the helper function, will do it right away

Reply:
Yeah, this function should be only called for elementary, so no point to keep any reference to Debian or check if Elementary

Reply:
We also need to check which version of ruby as old libssl is required only by old ruby.